index.html

```diff
@@ -408,6 +408,10 @@ <h3 id="downloads"> Downloads </h3>
     <td><img src='https://img.shields.io/amo/d/dustman.svg?maxAge=2592000' alt=''/></td>
     <td><code>https://img.shields.io/amo/d/dustman.svg</code></td>
   </tr>
+  <tr><th data-keywords='jetbrains plugin'> Jetbrains plugins: </th>
+    <td><img src='https://img.shields.io/jetbrainsplugins/org.intellij.scala/d.svg?maxAge=2592000' alt=''/></td>
+    <td><code>https://img.shields.io/jetbrainsplugins/PLUGIN_XML_ID/d.svg</code></td>
+  </tr>
```

Comment:
`index.html` has been removed in master. Could you merge master into this branch and remove it here too?